dev/MenuBar/MenuBar.cpp

```diff
@@ -18,8 +18,7 @@ MenuBar::MenuBar()
 }
 
 MenuBar::~MenuBar()
-{
-}
+= default;
 
```

Comment:
Is there a way to delete these instead? (i.e. not have the destructor or constructor if they're empty)?

Reply:
does not look like there is an option to automatically remove it. Again, we could write a custom one. http://clang.llvm.org/extra/clang-tidy/checks/modernize-use-equals-default.html

Reply:
I'll remove them manually for now.